.github/CODEOWNERS

```diff
@@ -0,0 +1,181 @@
+clang/** @llvm/pr-subscribers-clang
+clang/docs/DebuggingCoroutines.rst @llvm/pr-subscribers-coroutines
+clang/docs/StandardCPlusPlusModules.rst @llvm/pr-subscribers-modules
+clang/docs/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+clang/include/clang/AST/AbstractBasicReader.h @llvm/pr-subscribers-modules
+clang/include/clang/AST/AbstractBasicWriter.h @llvm/pr-subscribers-modules
+clang/include/clang/AST/AbstractTypeReader.h @llvm/pr-subscribers-modules
+clang/include/clang/AST/AbstractTypeWriter.h @llvm/pr-subscribers-modules
+clang/include/clang/AST/ODRHash.h @llvm/pr-subscribers-modules
+clang/include/clang/AST/PropertiesBase.td @llvm/pr-subscribers-modules
+clang/include/clang/AST/TypeProperties.td @llvm/pr-subscribers-modules
+clang/include/clang/Analysis/ @llvm/pr-subscribers-clang-analysis
+clang/include/clang/Basic/Module.h @llvm/pr-subscribers-modules
+clang/include/clang/CodeGen/** @llvm/pr-subscribers-clang-codegen
+clang/include/clang/Frontend/PrecompiledPreamble.h @llvm/pr-subscribers-modules
+clang/include/clang/Lex/ModuleLoader.h @llvm/pr-subscribers-modules
+clang/include/clang/Lex/ModuleMap.h @llvm/pr-subscribers-modules
+clang/include/clang/Serialization/* @llvm/pr-subscribers-modules
+clang/include/clang/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+clang/lib/AST/ODRHash.cpp @llvm/pr-subscribers-modules
+clang/lib/AST/StmtProfile.cpp @llvm/pr-subscribers-modules
+clang/lib/Analysis/ @llvm/pr-subscribers-clang-analysis
+clang/lib/Basic/Module.cpp @llvm/pr-subscribers-modules
+clang/lib/Basic/Targets/M68k.* @llvm/pr-subscribers-m68k
+clang/lib/CodeGen/** @llvm/pr-subscribers-clang-codegen
+clang/lib/CodeGen/CGCoroutine.cpp @llvm/pr-subscribers-coroutines
+clang/lib/CodeGen/CGDebugInfo.cpp @llvm/pr-subscribers-debug-info
+clang/lib/CodeGen/Targets/M68k.cpp @llvm/pr-subscribers-m68k
+clang/lib/Frontend/ModuleDependencyCollector.cpp @llvm/pr-subscribers-modules
+clang/lib/Frontend/PrecompiledPreamble.cpp @llvm/pr-subscribers-modules
+clang/lib/Lex/ModuleMap.cpp @llvm/pr-subscribers-modules
+clang/lib/Sema/SemaCoroutine.cpp @llvm/pr-subscribers-coroutines
+clang/lib/Sema/SemaModule.cpp @llvm/pr-subscribers-modules
+clang/lib/Serialization/* @llvm/pr-subscribers-modules
+clang/lib/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+clang/test/CXX/module/* @llvm/pr-subscribers-modules
+clang/test/CodeGenCoroutines/* @llvm/pr-subscribers-coroutines
+clang/test/Modules/* @llvm/pr-subscribers-modules
+clang/tools/scan-build/ @llvm/pr-subscribers-clang-static-analyzer
+clang/unittests/Serialization/* @llvm/pr-subscribers-modules
+clang/utils/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+compiler-rt/lib/memprof/* @llvm/pr-subscribers-pgo
+compiler-rt/lib/profile/* @llvm/pr-subscribers-pgo
+compiler-rt/test/memprof/* @llvm/pr-subscribers-pgo
+compiler-rt/test/profile/* @llvm/pr-subscribers-pgo
+flang/include/flang/Frontend @llvm/pr-subscribers-flang-driver
```

Comment:
We perhaps need a normalization that ensures every directory ends with a `/` :)

Reply:
Ah, sorry, this the and following 6 related to Flang should end with `/*`. Should've been more careful on Discourse.

Reply:
@tstellar This command `awk 'NF==2 { if ($1!~/\/$/ && $1!~/[*.]/) $1=$1"/"; print }'` can hopefully help you add `/` to directories (if not end with `/` && if no `*` or `.`, append `/`)

Reply:
I have a python script that generates this file, so it should be easy to fix, so basically anything not ending in `*` or `.` should have `/` appended?

Reply:
Perhaps:

* not ending with `/` or `*`
* does not have `.`

Append `/`